Failed to restore when running the Packager: Error NU1004: The package references have changed for net5.0

### General Description

Can't resolve the `Fluent.csproj` and the `Fluent.Desktop.csproj` when running the Packager with `dotnet run --onlybinaries`
### How To Reproduce?

1. Go to `WasabiWallet.Packager` folder
2. Run `dotnet run --onlybinaries`
3. See error


### Operating System

Win10

### Logs

```
Build succeeded.
    0 Warning(s)
    0 Error(s)

Time Elapsed 00:00:00.70
Process failed:
* Command: 'dotnet publish --configuration Release --force --output "E:\WalletWasabi\WalletWasabi.Fluent.Desktop\bin\dist\win7-x64" --self-contained true --runtime "win7-x64" --disable-parallel --no-cache /p:VersionPrefix=1.1.13 /p:DebugType=none /p:DebugSymbols=false /p:ErrorReport=none /p:DocumentationFile="" /p:Deterministic=true /p:RestoreLockedMode=true'
* Working directory: 'E:\WalletWasabi\WalletWasabi.Fluent.Desktop'
* Exit code: '1'
* Output:
  > Microsoft (R) Build Engine version 16.10.2+857e5a733 for .NET
  > Copyright (C) Microsoft Corporation. All rights reserved.
  >
  >   Determining projects to restore...
  >   Restored E:\WalletWasabi\WalletWasabi.Fluent.Generators\WalletWasabi.Fluent.Generators.csproj (in 291 ms).
  > E:\WalletWasabi\WalletWasabi.Fluent\WalletWasabi.Fluent.csproj : error NU1004: The package references have changed for net5.0. Lock file's package references: Avalonia:[0.10.10, ), Avalonia.Controls.DataGrid:[0.10.10, ), Avalonia.Diagnostics:[0.10.10, ), Avalonia.ReactiveUI:[0.10.10, ), Avalonia.Xaml.Behaviors:[0.10.10, ), OpenCvSharp4:[4.5.2.20210404, ), OpenCvSharp4.runtime.win:[4.5.2.20210404, ), System.Reactive:[5.0.0, ), System.Runtime:[4.3.1, ), project's package references: Avalonia:[0.10.10, ), Avalonia.Controls.DataGrid:[0.10.10, ), Avalonia.ReactiveUI:[0.10.10, ), Avalonia.Xaml.Behaviors:[0.10.10, ), OpenCvSharp4:[4.5.2.20210404, ), OpenCvSharp4.runtime.win:[4.5.2.20210404, ), System.Reactive:[5.0.0, ), System.Runtime:[4.3.1, ).The packages lock file is inconsistent with the project dependencies so restore can't be run in locked mode. Disable the RestoreLockedMode MSBuild property or pass an explicit --force-evaluate option to run restore to update the lock file. [E:\WalletWasabi\WalletWasabi.Fluent.Desktop\WalletWasabi.Fluent.Desktop.csproj]
  >   Failed to restore E:\WalletWasabi\WalletWasabi.Fluent\WalletWasabi.Fluent.csproj (in 22 ms).
  > E:\WalletWasabi\WalletWasabi.Fluent.Desktop\WalletWasabi.Fluent.Desktop.csproj : error NU1004: The project reference walletwasabi.fluent has changed. Current dependencies: Avalonia,Avalonia.Controls.DataGrid,Avalonia.ReactiveUI,Avalonia.Xaml.Behaviors,E:\WalletWasabi\WalletWasabi\WalletWasabi.csproj,OpenCvSharp4,OpenCvSharp4.runtime.win,System.Reactive,System.Runtime lock file's dependencies: Avalonia,Avalonia.Controls.DataGrid,Avalonia.Diagnostics,Avalonia.ReactiveUI,Avalonia.Xaml.Behaviors,OpenCvSharp4,OpenCvSharp4.runtime.win,System.Reactive,System.Runtime,WalletWasabi.The packages lock file is inconsistent with the project dependencies so restore can't be run in locked mode. Disable the RestoreLockedMode MSBuild property or pass an explicit --force-evaluate option to run restore to update the lock file.
  >   Failed to restore E:\WalletWasabi\WalletWasabi.Fluent.Desktop\WalletWasabi.Fluent.Desktop.csproj (in 13 ms).
  >   Restored E:\WalletWasabi\WalletWasabi\WalletWasabi.csproj (in 149 ms).
  >
Unhandled exception. System.InvalidOperationException: Process exited with unexpected exit code
   at WalletWasabi.Packager.Program.StartProcessAndWaitForExit(String command, String workingDirectory, String writeToStandardInput, String arguments, Boolean redirectStandardOutput) in WalletWasabi.Packager\Program.cs:line 563
   at WalletWasabi.Packager.Program.Publish() in WalletWasabi.Packager\Program.cs:line 273
   at WalletWasabi.Packager.Program.Main(String[] args) in WalletWasabi.Packager\Program.cs:line 78
   at WalletWasabi.Packager.Program.<Main>(String[] args)
```

### Wasabi Version
master

--- a/WalletWasabi.Fluent/WalletWasabi.Fluent.csproj
+++ b/WalletWasabi.Fluent/WalletWasabi.Fluent.csproj
@@ -23,7 +23,7 @@
 		<PackageReference Include="Avalonia" Version="$(AvaloniaVersion)" />
 		<PackageReference Include="Avalonia.Controls.TreeDataGrid" Version="0.10.10" />
 		<PackageReference Include="Avalonia.ReactiveUI" Version="$(AvaloniaVersion)" />
-		<PackageReference Include="Avalonia.Diagnostics" Version="$(AvaloniaVersion)" />
+		<PackageReference Include="Avalonia.Diagnostics" Version="$(AvaloniaVersion)" Condition="'$(Configuration)' == 'Debug'" />
 		<PackageReference Include="Avalonia.Xaml.Behaviors" Version="0.10.13.1" />
 		<PackageReference Include="Avalonia.Skia" Version="$(AvaloniaVersion)" />
 		<PackageReference Include="System.Drawing.Common" Version="6.0.0" />

--- a/WalletWasabi.Packager/Program.cs
+++ b/WalletWasabi.Packager/Program.cs
@@ -168,30 +168,35 @@ public static class Program
 		if (Directory.Exists(BinDistDirectory))
 		{
 			await IoHelpers.TryDeleteDirectoryAsync(BinDistDirectory).ConfigureAwait(false);
-			Console.WriteLine($"Deleted {BinDistDirectory}");
+			Console.WriteLine($"# Deleted {BinDistDirectory}");
 		}
 
+		Console.WriteLine($"# Run dotnet restore");
 		StartProcessAndWaitForExit("dotnet", DesktopProjectDirectory, arguments: "restore --locked-mode");
+
+		Console.WriteLine($"# Run dotnet clean");
 		StartProcessAndWaitForExit("dotnet", DesktopProjectDirectory, arguments: "clean --configuration Release");
 
-		var desktopBinReleaseDirectory = Path.GetFullPath(Path.Combine(DesktopProjectDirectory, "bin", "Release"));
-		var libraryBinReleaseDirectory = Path.GetFullPath(Path.Combine(LibraryProjectDirectory, "bin", "Release"));
+		string desktopBinReleaseDirectory = Path.GetFullPath(Path.Combine(DesktopProjectDirectory, "bin", "Release"));
+		string libraryBinReleaseDirectory = Path.GetFullPath(Path.Combine(LibraryProjectDirectory, "bin", "Release"));
+
 		if (Directory.Exists(desktopBinReleaseDirectory))
 		{
 			await IoHelpers.TryDeleteDirectoryAsync(desktopBinReleaseDirectory).ConfigureAwait(false);
-			Console.WriteLine($"Deleted {desktopBinReleaseDirectory}");
+			Console.WriteLine($"#Deleted {desktopBinReleaseDirectory}");
 		}
+
 		if (Directory.Exists(libraryBinReleaseDirectory))
 		{
 			await IoHelpers.TryDeleteDirectoryAsync(libraryBinReleaseDirectory).ConfigureAwait(false);
-			Console.WriteLine($"Deleted {libraryBinReleaseDirectory}");
+			Console.WriteLine($"# Deleted {libraryBinReleaseDirectory}");
 		}
 
 		var deterministicFileNameTag = IsContinuousDelivery ? $"{DateTimeOffset.UtcNow:ddMMyyyy}{DateTimeOffset.UtcNow.TimeOfDay.TotalSeconds}" : VersionPrefix;
 		var deliveryPath = IsContinuousDelivery ? Path.Combine(BinDistDirectory, "cdelivery") : BinDistDirectory;
 
 		IoHelpers.EnsureDirectoryExists(deliveryPath);
-		Console.WriteLine($"Binaries will be delivered here: {deliveryPath}");
+		Console.WriteLine($"# Binaries will be delivered here: {deliveryPath}");
 
 		string buildInfoJson = GetBuildInfoData();
 
@@ -209,7 +214,7 @@ public static class Program
 			if (!Directory.Exists(currentBinDistDirectory))
 			{
 				Directory.CreateDirectory(currentBinDistDirectory);
-				Console.WriteLine($"Created {currentBinDistDirectory}");
+				Console.WriteLine($"# Created {currentBinDistDirectory}");
 			}
 
 			string buildInfoPath = Path.Combine(currentBinDistDirectory, "BUILDINFO.json");
@@ -333,19 +338,19 @@ public static class Program
 				ZipFile.CreateFromDirectory(currentBinDistDirectory, zipFilePath);
 
 				await IoHelpers.TryDeleteDirectoryAsync(currentBinDistDirectory).ConfigureAwait(false);
-				Console.WriteLine($"Deleted {currentBinDistDirectory}");
+				Console.WriteLine($"# Deleted {currentBinDistDirectory}");
 
 				try
 				{
 					var drive = Tools.GetSingleUsbDrive();
 					var targetFilePath = Path.Combine(drive, zipFileName);
 
-					Console.WriteLine($"Trying to move unsigned zip file to removable ('{targetFilePath}').");
+					Console.WriteLine($"# Trying to move unsigned zip file to removable ('{targetFilePath}').");
 					File.Move(zipFilePath, targetFilePath, overwrite: true);
 				}
 				catch (Exception ex)
 				{
-					Console.WriteLine($"There was an error during copying the file to removable: '{ex.Message}'");
+					Console.WriteLine($"# There was an error during copying the file to removable: '{ex.Message}'");
 				}
 			}
 			else if (target.StartsWith("linux"))
@@ -363,7 +368,7 @@ public static class Program
 					continue;
 				}
 
-				Console.WriteLine("Create Linux .tar.gz");
+				Console.WriteLine("# Create Linux .tar.gz");
 				if (!Directory.Exists(publishedFolder))
 				{
 					throw new Exception($"{publishedFolder} does not exist.");
@@ -395,7 +400,7 @@ public static class Program
 
 				StartProcessAndWaitForExit("wsl", BinDistDirectory, arguments: arguments);
 
-				Console.WriteLine("Create Linux .deb");
+				Console.WriteLine("# Create Linux .deb");
 
 				var debFolderRelativePath = "deb";
 				var debFolderPath = Path.Combine(BinDistDirectory, debFolderRelativePath);
@@ -498,7 +503,7 @@ public static class Program
 				File.Move(oldDeb, newDeb);
 
 				await IoHelpers.TryDeleteDirectoryAsync(publishedFolder).ConfigureAwait(false);
-				Console.WriteLine($"Deleted {publishedFolder}");
+				Console.WriteLine($"# Deleted {publishedFolder}");
 			}
 		}
 	}
@@ -607,7 +612,7 @@ public static class Program
 		}
 		catch (Exception ex)
 		{
-			Console.WriteLine($"Process failed: '{ex}'.");
+			Console.WriteLine($"# Process failed: '{ex}'.");
 		}
 
 		return false;

--- a/azure-pipelines-linux.yml
+++ b/azure-pipelines-linux.yml
@@ -17,7 +17,7 @@ jobs:
     displayName: 'Restore'
     inputs:
       command: 'restore'
-      restoreArguments: '--locked-mode'
+      restoreArguments: '--force --locked-mode'
       feedsToUse: 'config'
       nugetConfigPath: 'NuGet.Config'
   - task: DotNetCoreCLI@2

--- a/azure-pipelines-osx.yml
+++ b/azure-pipelines-osx.yml
@@ -17,7 +17,7 @@ jobs:
     displayName: 'Restore'
     inputs:
       command: 'restore'
-      restoreArguments: '--locked-mode'
+      restoreArguments: '--force --locked-mode'
       feedsToUse: 'config'
       nugetConfigPath: 'NuGet.Config'
   - task: DotNetCoreCLI@2

--- a/azure-pipelines-windows.yml
+++ b/azure-pipelines-windows.yml
@@ -17,7 +17,7 @@ jobs:
     displayName: 'Restore'
     inputs:
       command: 'restore'
-      restoreArguments: '--locked-mode'
+      restoreArguments: '--force --locked-mode'
       feedsToUse: 'config'
       nugetConfigPath: 'NuGet.Config'
   - task: DotNetCoreCLI@2

